net-vpn/tor/tor-0.4.8.16.ebuild

```diff
@@ -186,7 +186,7 @@ src_install() {
 	readme.gentoo_create_doc
 
 	newconfd "${FILESDIR}"/tor.confd tor
-	newinitd "${FILESDIR}"/tor.initd-r9 tor
```

Comment:
This doesn't touch the live ebuild. It's also wrong to change the files installed by an ebuild in-place.